client/catapult/docs/BUILD-conan.md

```diff
@@ -62,24 +81,26 @@ Following instructions should work on Mac, Linux (Ubuntu 20.04) and Windows.
 While Conan will be building and installing packages, you might want to go for a ☕ (or lunch),
 as this will probably take *a bit*.
 
+### Install conan sources repo and get catapult source code (any OS)
 ```sh
-conan remote add nemtech https://conan.symbol.dev/artifactory/api/conan/catapult
-
-git clone https://github.com/symbol/symbol.git
-cd symbol/client/catapult
+$> conan remote add nemtech https://conan.symbol.dev/artifactory/api/conan/catapult
```

Comment:
As you pointed out is only a stylistic reason.
As a side note I generally do not encourage copy paste of the command line as it may happen some arguments could be derived from specific local environment variables or paths.

In any case I'm removing those to be consistent.